Philips.CodeAnalysis.MaintainabilityAnalyzers/Maintainability/AvoidTodoCommentsAnalyzer.cs

```diff
@@ -38,12 +39,41 @@ public override void Analyze()
 			foreach (SyntaxTrivia comment in comments)
```

Comment:
if (!comments.Any) return
seems to be adding no value, as it would safely pass through the foreach loop anyway.